io.catenax.item_stock/2.0.0/ItemStock.ttl

```diff
@@ -0,0 +1,174 @@
+    samm:properties (:quantityOnAllocatedStock :stockLocationBPNS :isBlocked :stockLocationBPNA).
+:supplierOrderId a samm:Property;
+    samm:preferredName "Supplier Order ID"@en;
+    samm:description "This is the order number on the supplier side that is given by e.g., the supplier's enterprise resource planning system."@en;
+    samm:characteristic :OrderIdCharacteristic;
+    samm:exampleValue "M-Nbr-4711".
+:customerOrderId a samm:Property;
+    samm:preferredName "Customer Order ID"@en;
+    samm:description "This is the order number on the customer side that can be given by e.g., the customer's enterprise resource planning system."@en;
+    samm:characteristic :OrderIdCharacteristic;
+    samm:exampleValue "C-Nbr-4711".
+:customerOrderPositionId a samm:Property;
+    samm:preferredName "Customer Order Position ID"@en;
+    samm:description "This is an identifier for the position of an order defined by the customer. It's not a technical identifier, such as an UUID. This identifier is used within the business context when talking about the position."@en;
+    samm:characteristic :OrderPositionIdCharacteristic;
+    samm:exampleValue "PositionId-01".
+:quantityOnAllocatedStock a samm:Property;
+    samm:preferredName "Quantity on Allocated Stock"@en;
+    samm:description "This is the quantity of the material on stock. In case of stocks \"from a supplier ready to be shipped to a certain customer\", the stock may refer to an order position of a customer."@en;
+    samm:characteristic :QuantityCharacteristic.
+:stockLocationBPNS a samm:Property;
+    samm:preferredName "BPNS of Stock Location"@en;
+    samm:description "BPNS identifying the site the stock is located at. One stock has always one location."@en;
+    samm:characteristic ext-number:BpnsTrait;
+    samm:exampleValue "BPNS1234567890ZZ";
+    samm:see <https://catena-x.net/fileadmin/user_upload/Standard-Bibliothek/Update_PDF_Maerz/5_BPDM/CX_-_0010_BUSINESS_PARTNER_NUMBER_PlatformCapabilityBPDM_v_1.0.1.pdf>.
+:isBlocked a samm:Property;
+    samm:preferredName "Is Blocked"@en;
+    samm:description "A set of finished items that are located on the blocked stock area, quarantine area, etc. waiting to be released.\n\nExample: Stock is in review and not released due to customer complaint regarding a recent batch."@en;
+    samm:characteristic :IsBlockedCharacteristic;
+    samm:exampleValue "false"^^xsd:boolean.
+:stockLocationBPNA a samm:Property;
+    samm:preferredName "BPNA of Stock Location"@en;
+    samm:description "BPNA identifying the address the stock is located at. One stock has always one location."@en;
+    samm:characteristic ext-number:BpnaTrait;
+    samm:exampleValue "BPNA1234567890";
```

Comment:
invalid example value. perhaps you can change it to something like BPNA1234567890ZZ